src/Symfony/Component/Messenger/Bridge/AmazonSqs/Transport/AmazonSqsFifoStamp.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Messenger\Bridge\AmazonSqs\Transport;
+
+use Symfony\Component\Messenger\Stamp\NonSendableStampInterface;
+
+final class AmazonSqsFifoStamp implements NonSendableStampInterface
+{
+    private $messageGroupId;
+
+    private $messageDeduplicationId;
+
+    public function __construct(string $messageGroupId, string $messageDeduplicationId)
+    {
+        $this->messageGroupId = $messageGroupId;
+        $this->messageDeduplicationId = $messageDeduplicationId;
+    }
+
+    public function getAttributes(): array
```

Comment:
This method sounds coupled with the connection implementation. And is not easy for users to fetch properties.
I'd rather used a `getMessageGroupId` and `getMessageDeduplicationId`.
WDYT?